README.md

```diff
@@ -29,27 +37,22 @@ or if you want to rebuild the images
 
 `docker-compose up --build -d`
```

Comment:
maybe we should use `docker compose` now?

Reply:
Not sure how would it go, opened another issue for this.